Jenkinsfile

```diff
@@ -315,6 +315,16 @@ try {
                 Utils.markStageSkippedForConditional("tpcdsQueryPlansAndVerification")
               }
             }
+          }, jobQueryPlans: {
+            stage("jobQueryPlans") {
+              if (env.BRANCH_NAME == 'master' || full_ci) {
+                sh "mkdir -p query_plans/job && ./clang-release/hyriseBenchmarkJoinOrder -r 1 --visualize && ./scripts/plot_operator_breakdown.py ./clang-release/ && mv operator_breakdown.pdf query_plans/job && mv *.svg query_plans/job"
+                archiveArtifacts artifacts: 'query_plans/job/*.svg'
+                archiveArtifacts artifacts: 'query_plans/job/operator_breakdown.pdf'
```

Comment:
For consistency, consider calling hyriseBenchmarkJoinOrder and scripts from query_plans/job.

Reply:
Yeah, I wanted to do that but `hyriseBenchmarkJoinOrder` checks whether the binary is called from the main folder (because it internally calls the setup_imdb.py script). We could change that behavior but I don't consider it worth it.

Reply:
I will add a comment though.

Reply:
Got it. Makes sense then.